src/containers/note/toolbar.tsx

```diff
@@ -92,19 +118,71 @@ function Toolbar({ editor }: ToolbarProps) {
       ariaLabel: "H5",
       isActive: editor.isActive("heading", { level: 5 }),
     },
+    // 링크 버튼
+    {
+      onClick: openLinkModal,
+      IconComponent: LinkIcon,
+      ariaLabel: "Link",
+      isActive: false,
+    },
   ];
 
+  const linkInput = (
+    <input
+      type="text"
+      value={inputLink}
+      onChange={handleInputChange}
+      placeholder="https://"
+      className="h-44 w-full truncate rounded-12 bg-slate-50 p-5 text-sm md:h-48 md:text-base"
+    />
+  );
+
+  const confirmButton = (
+    <Button
+      variant="solid"
+      shape="square"
+      size="lg"
+      className="w-full"
+      onClick={closeLinkModal}
+    >
+      확인
+    </Button>
+  );
+
   return (
-    <div className="container-width fixed bottom-24 flex h-44 rounded-22 border-slate-200 bg-white p-2 pl-16 shadow">
-      {buttonConfigs.map((btn) => (
+    <div
+      className="container-width fixed bottom-24 flex h-44 items-center justify-center rounded-22 border-slate-200 bg-white p-2 pl-16 shadow"
+      style={{ justifyContent: "space-between" }}
+    >
+      <div style={{ display: "flex" }}>
+        {buttonConfigs.slice(0, -1).map((btn) => (
+          <ToolbarButton
+            key={btn.ariaLabel}
+            onClick={btn.onClick}
+            IconComponent={btn.IconComponent}
+            ariaLabel={btn.ariaLabel}
+            isActive={btn.isActive}
+          />
+        ))}
+      </div>
+      <div style={{ marginRight: "16px" }}>
         <ToolbarButton
-          key={btn.ariaLabel}
-          onClick={btn.onClick}
-          IconComponent={btn.IconComponent}
-          ariaLabel={btn.ariaLabel}
-          isActive={btn.isActive}
+          onClick={buttonConfigs[buttonConfigs.length - 1].onClick}
+          IconComponent={buttonConfigs[buttonConfigs.length - 1].IconComponent}
+          ariaLabel={buttonConfigs[buttonConfigs.length - 1].ariaLabel}
+          isActive={buttonConfigs[buttonConfigs.length - 1].isActive}
         />
-      ))}
+      </div>
+      {isLinkModalOpen && (
+        <Modal.Root open={isLinkModalOpen} onOpenChange={setLinkModalOpen}>
+          <Modal.Content className="h-268 w-311 md:h-272 md:w-520">
```

Comment:
height 관련 클래스 때문에 불필요한 스크롤바가 생겨서 없애는 편이 좋을 것 같아요!

![image](https://github.com/user-attachments/assets/6cfe921b-1827-4685-be97-34d85b74daff)

```suggestion
          <Modal.Content className="w-311md:w-520">
```